Source/ee/IPU.cpp

```diff
@@ -734,7 +734,9 @@ void CIPU::COUTFIFO::RequestGrow(unsigned int size)
 	while(m_alloc <= (size + m_size))
 	{
 		m_alloc += GROWSIZE;
-		m_buffer = reinterpret_cast<uint8*>(realloc(m_buffer, m_alloc));
```

Comment:
ah I see... and subtracting m_alloc would cause a potential infinite loop. but won't sending old buffer better than sending null back?